batch/create-batch-tables-pod.yaml

```diff
@@ -20,6 +20,15 @@ spec:
          mysql --defaults-extra-file=/sql-config/sql-config.cnf < ./create-batch-tables.sql
          python3 ./insert-batch-globals.py {{ scope }}
          mysql --defaults-extra-file=/sql-config/sql-config.cnf < ./create-billing-projects.sql
+         {% if scope == 'dev' %}
+         mysql --defaults-extra-file=/sql-config/sql-config.cnf <<'EOF'
+       INSERT INTO `billing_projects` (`name`)
+       VALUES ("hail");
+
+       INSERT INTO `billing_project_users` (`billing_project`, `user`)
+       VALUES ("hail", "{{ default_ns.name }}");
+       EOF
+         {% endif %}
```

Comment:
This is about to go away when my database migrations PR goes in. Can you remove it from here and I'll add it there?  This doesn't feel like quite the right place for this, either, but I don't know where it should go.